docs/metrics.md

```diff
@@ -0,0 +1,171 @@
+# METRICS
+
+## Data Analysis
```

Comment:
Yes to both. @TanviHacks can say for sure, but I know the ultimate goal is to launch the feature into release. The intermediate goal is to learn how users interact with the UI so we know what parts of the feature are most useful. (E.g., do users sort? show/hide? move tabs to new window?)